debian/rules

```diff
@@ -107,8 +104,8 @@ endif
 	    -DCONC_DEFAULT_CHARSET=utf8mb4 \
 	    -DPLUGIN_AWS_KEY_MANAGEMENT=NO \
 	    -DPLUGIN_COLUMNSTORE=NO \
-	    -DIGNORE_AIO_CHECK=YES \
-	    -DWITH_URING=YES \
+	    -DIGNORE_AIO_CHECK=ON \
```

Comment:
Is this wise to-do as this haven't been enabled before?

Reply:
This just changes `YES` to `ON` to unify the terms. There are no functional changes.

Reply:
There is this `replace_uring_with_aio` in autobake-debs.sh which currently turns on IGNORE_AIO_CHECK and WITH_URING and it's done after Debian Buster (EOL it does not need to be fixed) and Ubuntu Bionic will EOL this year but next release is build against it.
Bionic needs to be addressed